example3/Microsoft.Diagnostics.Metric/MetricLabel.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+namespace Microsoft.Diagnostics.Metric
+{
+    public class MetricLabel
+    {
+        static private (string name, string value)[] emptyLabel = {};
+
+        static private MetricLabel defaultLabel = new MetricLabel();
+
+        static public MetricLabel DefaultLabel
+        {
+            get => defaultLabel;
+        }
+
+        /// <summary>
+        /// Return Array of Tuple&lt;Key,Value&gt;.
+        /// </summary>
+        public virtual (string name, string value)[] GetLabels()

```

Comment:
@noahfalk Any concerns with use of Value Tuple?  I think we can use array of Value Tuple for our "LabelSets" that is a good compromise between usability vs performance.

Reply:
For API usability .NET team typically recommends using a named struct with properties. It will have the same performance characteristics as the tuple.